app/backend/controllers/auth.js

```diff
@@ -0,0 +1,9 @@
+module.exports.authenticationBinder = () => {
+  return (req, res, next) => {
+    if(!req.session['user']) {
```

Comment:
A simple comment on why that middleware is written and where it would be used would be very nice.